yorkie/backend/sync/etcd/config.go

```diff
@@ -0,0 +1,43 @@
+package etcd
+
+import (
+	"errors"
+	"fmt"
+	"time"
+)
+
+var (
+	//ErrEmptyEndPoints occurs when the endpoints in the config is empty.
+	ErrEmptyEndPoints = errors.New("length of etcd endpoints must be greater than 0")
+	// ErrEmptyUserName occurs when the username is empty.
+	ErrEmptyUserName = errors.New("etcd username cannot be nil or empty")
+	// ErrEmptyPassword occurs when the password is empty.
+	ErrEmptyPassword = errors.New("etcd password cannot be nil or empty")
+)
+
+// Config is the configuration for creating a Client instance.
+type Config struct {
+	Endpoints      []string      `json:"Endpoints"`
+	DialTimeoutSec time.Duration `json:"DialTimeoutSec"`
```

Comment:
Yes. I agree we don't have to append `XXXSec` suffixes. Because with CLI `DurationVar` flags we can set any duration units.

> Duration flags take any input valid for time.ParseDuration. from https://morioh.com/p/8a1165374aa6

Reply:
Thank you for telling me! We will proceed after merging the current pull request.